packages/hash/api/src/auth/index.ts

```diff
@@ -99,7 +105,7 @@ const setupAuth = (params: {
   // eslint-disable-next-line @typescript-eslint/no-misused-promises -- https://github.com/DefinitelyTyped/DefinitelyTyped/issues/50871
   app.use(async (req, _res, next) => {
     const kratosSession = await publicKratosSdk
```

Comment:
How about using also including the `xSessionToken` property? it would enable API clients (using the ory API self-service login) to use HASH, notably the graphql endpoint.

> Suggestion taken from https://www.ory.sh/docs/identities/sign-in/check-session-token-cookie-api

```suggestion
    const authHeader = req.headers.authorization;
    const hasAuthHeader = authHeader.startsWith("Bearer ");
    const sessionToken = hasAuthHeader ? authHeader.slice(7, authHeader.length) : null;

    const kratosSession = await publicKratosSdk
```

Reply:
Thank you so much! I've applied this in ac3955992d57c3cb7e702def5e1b965c6cad2854 <3